pkg/tsdb/stackdriver/stackdriver_test.go

```diff
@@ -208,6 +208,95 @@ func TestStackdriver(t *testing.T) {
 
 		})
 
+		Convey("Parse queries from frontend and build Stackdriver API queries", func() {
+			fromStart := time.Date(2018, 3, 15, 13, 0, 0, 0, time.UTC).In(time.Local)
+			tsdbQuery := &tsdb.TsdbQuery{
+				TimeRange: &tsdb.TimeRange{
+					From: fmt.Sprintf("%v", fromStart.Unix()*1000),
+					To:   fmt.Sprintf("%v", fromStart.Add(34*time.Minute).Unix()*1000),
+				},
+				Queries: []*tsdb.Query{
+					{
+						Model: simplejson.NewFromAny(map[string]interface{}{
+							"queryType": metricQueryType,
+							"metricQuery": map[string]interface{}{
+								"metricType": "a/metric/type",
+								"view":       "FULL",
+								"aliasBy":    "testalias",
+								"type":       "timeSeriesQuery",
```

Comment:
It would be nice to include a `groupBys` field and check the `queries[0].Params["aggregation.groupByFields"]` and `queries[0].GroupBys`.

Reply:
👍